src/reynir/Greynir.grammar

```diff
@@ -3223,46 +3359,62 @@ NlFrumlagLeppur → 'það:pfn'_hk_nf_et
 
 NlAndlagÞað/fall → 'það:pfn'_hk_et/fall
 
-NlFrumlag/fall → Nl/fall | NlSkýringarsetning
+SkýringarsetningFrumlag → NlSkýringarsetning
```

Comment:
See comment regarding `$tag(keep)` above - to ensure that nonterminals are not optimized away.

Reply:
Done